### ISSUE TITLE
Audio Mute Icon not properly positioned

### ISSUE THREAD

**Issue by ahattangadi** [black7375/Firefox-UI-Fix#463]
**Describe the bug**
The audio mute icon, covers a part of the favicon.

**Expected behavior**
The audio mute icon should cover the text, rather than the favicon.

**Screenshots**
![image](https://user-images.githubusercontent.com/69454819/188302966-30d467e2-c4bb-4018-a0c4-b4d63ba79d08.png)

**Environment:**

 - Distribution
   - [ ] [Original Lepton](https://github.com/black7375/Firefox-UI-Fix)
   - [x] [Lepton's photon style](https://github.com/black7375/Firefox-UI-Fix/tree/photon-style)
   - [ ] [Lepton's proton style](https://github.com/black7375/Firefox-UI-Fix/tree/proton-style)
 - Firefox Version: 
 - OS:
   - [ ] Linux
   - [x] Windows
   - [ ] Mac
   - [ ] Other: 
 - Theme:
   - [ ] Light
   - [x] Dark
   - [ ] Alpenglow (System: Light)
   - [ ] Alpenglow (System: Dark)
   - [ ] System Default
     - GTK Theme Name (Linux only): 
   - [ ] Other: (e.g. [Photon Colors](https://addons.mozilla.org/en-US/firefox/addon/photon-colors/), [Google Chrome Dark](https://addons.mozilla.org/en-US/firefox/addon/google-chrome-dark/), [Fairytale Of Nature](https://addons.mozilla.org/en-US/firefox/addon/fairytale-of-nature/))

<details><summary><strong><code>user.js</code> setup</strong></summary><br>

```javascript
I've not changed anything - just run the powershell script :)
```
</details>
 
**Additional context**

**Comment by black7375** [comment]
Do you have any screenshots you want?
It's not the default, but I can add it as an option later.

**Comment by black7375** [comment]
I haven't worked on document yet, but your proposal is included in 8.0.0.
- `userChrome.tab.sound_with_favicons.on_center` to `true`
